src/RapidRequest.js

```diff
@@ -9,8 +9,13 @@ const fetchRequests = async ({ baseUrl, locationSecret, locationKey, locationCon
   let requests = [];
 
   const headers = {
-    "x-location-secret": locationSecret,
+    "x-location-secret": locationSecret
   };
+
+  if (process.env.FORWARD_URL) {
+    headers["x-forwarded-for"] = process.env.FORWARD_URL;
```

Comment:
x-forwarded-for is always an IP address, never a URL

Reply:
good point!